examples/alert.tf

```diff
@@ -46,10 +46,34 @@ resource "humio_alert" "example_alert_with_description" {
     humio_action.example_email_subject.action_id,
   ]
 
-  link_url             = "http://localhost:8080/humio/search?query=count()&live=true&start=24h&fullscreen=false"
   labels               = ["terraform", "ops"]
   throttle_time_millis = 300000
   enabled              = true
   query                = "count()"
-  start                = "24h"
+  start                = "1d"
+}
+
+resource "humio_alert" "example_alert_with_user_owner" {
+  repository = humio_action.example_email_body.repository
+  name       = "example_alert_with_user_owner"
+
+  throttle_time_millis = 300000
+  enabled              = true
+  query                = "count()"
+  start                = "1d"
+
+  query_ownership_type = "User"
```

Comment:
```suggestion
  query_ownership_type = "User"
  run_as_user_id       = "John Doe"
```

?

Reply:
> I've added no example using `run_as_user_id` as this requires a real user ID, a `[a-Z0-9]{24}`-looking string.

The above would fail when trying to deploy it, so I figured it was better to just not include any examples of it. We could add the line as commented out, and write something like 

    # run_as_user_id       = "XXXXXXXXXXXXXXXXXXXXXXXX"

Thoughts?